tests/conftest.py

```diff
@@ -12,6 +12,17 @@
 pytest_plugins = ['aiida.manage.tests.pytest_fixtures']  # pylint: disable=invalid-name
 
 
+@pytest.fixture(scope='session')
+def filepath_tests():
+    """Return the absolute filepath of the `tests` folder.
+
+    .. warning:: if this file moves with respect to the `tests` folder, the implementation should change.
+
+    :return: absolute filepath of `tests` folder which is the basepath for all test resources.
+    """
+    return os.path.dirname(__file__)
```

Comment:
I think this should be ``os.path.dirname(os.path.abspath(__file__)``. It's a bit of an edge case (which might never happen here), but e.g. when I run a script at the root of the filesystem (``/script.py``) ``os.path.dirname(__file__)`` is an empty string, whereas ``os.path.dirname(os.path.abspath(__file__))`` is ``/``.

Reply:
Yeah, but this would be with respect of the `conftest.py` file, which will never be located in root, no? So this will never occur

Reply:
Yes, but I'm not 100% sure if the ``__file__`` will always be an absolute path.

Reply:
Sorry the example was kinda bad, that was just to show that the `os.path.dirname(__file__)` idiom doesn't automatically give the correct abspath in all cases.

Reply:
ah ok, fair enough then it makes sense, I will change it

Reply:
Just realized that this also happens when the directory is _not_ root - ``__file__`` appears to be relative to the CWD. I'm guessing this worked because it's converted to ``abspath`` in the cases where the ``filepath_tests`` is used. That can probable be removed.

Reply:
wait, I don't understand. What can be removed?

Reply:
If we add ``os.path.abspath`` here, it can be removed in the calling code (see my other comments).